simplefs/simplefs.go

```diff
@@ -430,6 +523,37 @@ func (k *SimpleFS) SimpleFSReadList(_ context.Context, opid keybase1.OpID) (keyb
 	return lr, nil
 }
 
+func recursiveByteAndFileCount(fs billy.Filesystem) (
```

Comment:
It's still not easy. As I mentioned yesterday, we'll need to crawl the full subtree to get folder size, so it's not a cheap operation. When you're copying something recursively, then you already have to do that so it's not as big a deal.

We'll need a ticket to cache a `RecursiveSize` field in `DirEntry`.

Reply:
We don't have to do that automatically. We can only do it as user clicks a button or something, so it's not like it slows down the app unnecessarily.

> We'll need a ticket to cache a RecursiveSize field in DirEntry.

For discussion purposes and not near future planning: do you mean cache it in the directory block? Do we update it as we update anything under the directory, or only lazily update it as it's requested?

Reply:
I mean `DirEntry`, but directory block would work too (and would be less disruptive to folder capacity). Yes, we update these totals as we crawl the tree on any writes, since we have to crawl the tree at that point anyway.

Reply:
I see. That makes sense. Sounds like a good idea!